hr_job_category/__manifest__.py

```diff
@@ -3,7 +3,7 @@
 
 {
     "name": "HR Job Employee Categories",
-    "version": "15.0.1.0.1",
+    "version": "17.0.1.0.1",
```

Comment:
we shall specify version: `"version": "17.0.1.0.0"`

Reply:
indeed 👍🏻